types/restify/index.d.ts

```diff
@@ -1495,13 +1495,13 @@ export namespace plugins {
      * Listens to the server's after event and emits information about that request (5.x compatible only).
      *
      * ```
-     * server.on('after', plugins.metrics( (err, metrics) =>
+     * server.on('after', plugins.metrics({ server }, (err, metrics, req, res, route) =>
      * {
      *    // metrics is an object containing information about the request
      * }));
      * ```
      */
-    function metrics(opts: { server: Server }, callback: (options: MetricsCallback) => any): (...args: any[]) => void;
+    function metrics(opts: { server: Server }, callback: MetricsCallback): (...args: any[]) => void;
```

Comment:
This doesn't seem right. http://restify.com/docs/plugins-api/#metrics according to this callback is of type Function so it needs to be function type which metricsCallback isn't?

Reply:
@sheetalkamat you are right. However the current implementation is also wrong. 

I'll take another stab at fixing it correctly.

Reply:
I adjusted the type of the `MetricsCallback` to match.